handlers/error.go

```diff
@@ -38,6 +40,7 @@ const (
 	ErrConvertToSpecCode        = "2028"
 	ErrFetchSMIResultsCode      = "2029"
 	ErrFormFileCode             = "2030"
+	ErrDownloadFileCode         = "replace me"
```

Comment:
Hey @metonymic-smokey I think it would be better if we can make all the error codes to "replace_me" so that all get a int code serially.

Reply:
Done @sayantan1413 !
Thanks!